deaddrop/urls.py

```diff
@@ -29,4 +29,4 @@
 ]
 
 # Files will be served statically.
-urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)

```

Comment:
consider either two lines of `urlpatterns += ...` or just adding them to the list above for consistency